_opensearch/install/tar.md

```diff
@@ -7,144 +7,455 @@ nav_order: 50
+   # x64
+   tar -xvf opensearch-{{site.opensearch_version}}-linux-x64.tar.gz
+   
+   # ARM64
+   tar -xvf opensearch-{{site.opensearch_version}}-linux-arm64.tar.gz
    ```
 
-   If you configure the security plugin for production use (or disable it), you can run OpenSearch using `./bin/opensearch`.
-
-1. Open a second terminal session, and send requests to the server to verify that OpenSearch is up and running:
+## Configure Important System Settings
 
+Before launching OpenSearch you should review some [important system settings]({{site.url}}{{site.baseurl}}/opensearch/install/important-settings/){:target='\_blank'}.
+1. Disable memory paging and swapping performance on the host to improve performance.
    ```bash
-   curl -XGET https://localhost:9200 -u 'admin:admin' --insecure
-   curl -XGET https://localhost:9200/_cat/plugins?v -u 'admin:admin' --insecure
+   sudo swapoff -a
    ```
+1. Increase the number of memory maps available to OpenSearch.
+   ```bash
+   # Edit the sysctl config file
+   sudo vi /etc/sysctl.conf
 
+   # Add a line to define the desired value
+   # or change the value if the key exists,
+   # then save your changes.
+   vm.max_map_count=262144
 
-## Configuration
+   # Reload the kernel parameters using sysctl
+   sudo sysctl -p
 
-You can modify `config/opensearch.yml` or specify environment variables as arguments using `-E`:
+   # Verify that the change was applied by checking the value
+   cat /proc/sys/vm/max_map_count
+   ```
 
-```bash
-./opensearch-tar-install.sh -Ecluster.name=opensearch-cluster -Enode.name=opensearch-node1 -Ehttp.host=0.0.0.0 -Ediscovery.type=single-node
-```
+## (Optional) Test OpenSearch
+
+Before proceeding you should test your installation of OpenSearch. Otherwise, it can be difficult to determine whether future problems are due to installation issues or custom settings you applied after installation. There are two quick methods for testing OpenSearch at this stage:
 
-For other settings, see [Important settings]({{site.url}}{{site.baseurl}}/opensearch/install/important-settings/).
+1. **(Security Enabled)** Apply a generic configuration using the demo security script included in the tar archive.
+1. **(Security Disabled)** Manually disable the security plugin and test the instance before applying your own custom security settings.
 
+The demo security script will apply a generic configuration to your instance of OpenSearch. This configuration defines some environment variables and also applies self-signed TLS certificates. If you would like to configure these yourself, see [Setup OpenSearch in your environment](#setup-opensearch-in-your-environment).
 
-### (Optional) Set up Performance Analyzer
+If you only want to verify that the service is properly configured and will respond to API requests, and you intend to configure security settings yourself, then you may want to disable the security plugin and launch the service without encryption or authentication.
```

Comment:
Delete comma after "requests"

Reply:
I went ahead and removed the "and responds to API requests" bit because it was extraneous and simplifies the sentence.